src/main/java/website/frontrow/board/Bubble.java

```diff
@@ -77,6 +78,16 @@ public void applyGravity()
     {
         // Ignore gravity.
     }
+    
+    /**
+     * Return the bubble speed multiplier.
+     * @return the bubble speed multiplier
+     */
+    @Override
+    public double getUnitSpeedMultiplier()
```

Comment:
Rename to getSpeedMultiplier please, we already know we're getting it from a Unit.

< naming